pkg/server/api.go

```diff
@@ -60,12 +60,23 @@ func NewAPIServer(a *APIHandler, p int, is bool, c, k string) *APIServer {
 
 // Serve launches the API Server.
 func (a *APIServer) Serve() {
```

Comment:
can these code changes be broken out into a separate commit from the go upgrade?

Reply:
I literally just pushed that commit :)

Reply:
I updated the `machines-config-daemon.spec` too -- it would be required.

Reply:
That file is dead btw, the MCD isn't part of RHCOS anymore.

Reply:
I *thought* the `spec` file was dead, but wasn't 100% sure.

Reply:
>I literally just pushed that commit :)

great minds!

Reply:
yeah as Colin said we no longer need and build machine-config-daemon rpm package. Even if someone wants to build m-c-d rpm package it will build fine with 1.13 because the current 1.15 requirement is added mainly to MCS.